ampd/src/queue/queued_broadcaster.rs

```diff
@@ -236,17 +235,19 @@ mod test {
             .expect_broadcast()
             .once()
             .returning(move |msgs| {
-                assert!(msgs.len() == tx_count);
+                assert_eq!(msgs.len(), tx_count);
 
                 Ok(TxResponse::default())
             });
 
-        let queued_broadcaster = QueuedBroadcaster::new(
-            broadcaster,
-            batch_gas_limit,
-            tx_count,
-            Duration::from_secs(5),
-        );
+        let mut desired_interval = interval(Duration::from_secs(5));
+        select! {
+            _ = desired_interval.tick() => {
+            },
```

Comment:
```suggestion
            _ = desired_interval.tick() => {}
```
or does the formatter choose this format?

Reply:
I don't understand this `select!` at all. Why not just `broadcast_interval.tick().await;`?